### PR TITLE
My first file

### PR DESCRIPTION
**Pull request by mjibaja**
My first pull

### CODE REVIEW

File: _pins/mjibaja.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: mjibaja
+latitude: 	12°0′S
+longitude: 77°0′O
```
Comment (hectorsector) [inline _pins/mjibaja.json]: @mjibaja enter the coordinates using decimals, not degrees. You can use the NASA site linked on the class instructions to find it.

**Comment by githubteacher** [comment]
@mjibaja it looks like you still have some work to do on your Pull Request. Here are the instructions for how to create your file:

screen-shot-2016-10-11-at-10 01 58-am

Once you have finished all of these steps, please mention @hectorsector so he can approve your changes!

**Comment by lucianoh** [comment]
Try fixing your coordinates by using the nasa link

**Comment by returntrip** [comment]
Please change the coordinates to decimal format! :loudspeaker:

**Comment by Player100** [comment]
:blue_heart: you should change the coordinates to decimal format

**Comment by justcallmetrell** [comment]
Not that I have to add to the solution, but please change the format of the coordinates to decimal format. :smile: :laughing:

**Pull request merged**